src/libopensc/pkcs15-openpgp.c

```diff
@@ -69,6 +70,7 @@ static const pgp_pin_cfg_t	pin_cfg_v2[3] = {
 	{ "User PIN (sig)", 0x01, PGP_USER_PIN_FLAGS,  6, 0 },	// used for PSO:CDS
 	{ "Admin PIN",      0x03, PGP_ADMIN_PIN_FLAGS, 8, 2 }
 };
+// clang-format on
 
 
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/pkcs15-openpgp.c -->